packages/pluggableWidgets/accordion-web/src/ui/accordion-main.scss

```diff
@@ -0,0 +1,48 @@
+$border-color: #ced0d3;
+
+.widget-accordion {
+    background-color: #ffffff;
```

Comment:
Can we use a variable for the background too?

Reply:
Yes, I also added variables for the other colors that are used in the functional styles.